scripts/AGOV-EN/c101202085.lua

```diff
@@ -0,0 +1,51 @@
+--coded by Lyris
+--Wattsychic Fighting Porter

```

Comment:
This one is committed under https://github.com/Fluorohydride/ygopro-pre-script/pull/1187

Reply:
You submitted before him. Please ignore the above comment